README.md

```diff
@@ -316,7 +318,39 @@ If you use arbitration graphs in your research, we would be pleased if you cite
   doi = {10.48550/arXiv.2411.10170},
   url={https://arxiv.org/abs/2411.10170}, 
 }
+```
+{% endraw %}
+
+
+<details>
+<summary>Earlier publications</summary>
+
+A safety concept that extends Arbitration Graphs with behavior verification and fallback layers in the context of automated driving has been proposed by Piotr Spieker (née Orzechowski) in his PhD thesis.
```

Comment:
I'd add horizontal dividers between the publications to distinguish more easily which part belongs where. See also the analogous suggestions below.

```suggestion
---
A safety concept that extends Arbitration Graphs with behavior verification and fallback layers in the context of automated driving has been proposed by Piotr Spieker (née Orzechowski) in his PhD thesis.
```